test/recipe-test.js

```diff
@@ -1,7 +1,8 @@
 import { expect } from 'chai';
```

Comment:
Not sure why, but it's not allowing line comments for half of the file. Anyway.

There are two tests coming up failing here:
- should initialize with an array of ingredients
- should calculate the total cost of all of the ingredients

The first of these is failing since we changed the test data. There is no longer a name in the object, and it is still expecting one. We will want to change that soon.

The second is a formatting issue when we calculate the cost. Right now we're finishing with a number, and it's expecting the string with the `$` sign. We will want to decide how we will be displaying this to the dom, then test and implement accordingly.